src/pages/index.js

```diff
@@ -71,6 +60,31 @@ const cardSubmitBtn = editCardModal.querySelector(".modal__save-btn");
 const cardTemplate = document.querySelector("#card-template");
 const cardsList = document.querySelector(".cards__list");
 
+let selectedCard, selectedCardId;
+
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "6f4ae429-4ad9-43f6-abd7-d143d087e687",
+    "Content-Type": "application/json",
+  },
+});
+
+api
+  .getCardsAndUserInfo()
+  .then(([cards, userInfo]) => {
+    cards.forEach((item) => {
+      const cardElement = getCardElement(item);
+      cardsList.prepend(cardElement);
+    });
+
```

Comment:
* When I reload the page, the old `avatar` appears again

Please, set the `avatar` on load